projects/convenience.md

```diff
@@ -0,0 +1,91 @@
+# Convenience Project
+
+## Background and description
+
+Convenience as a SIG was initially proposed by [Ted
+Young](https://github.com/tedsuo) back in 2021. It was postponed to prioritize
+completion of signal specs and stabilization. A document was written at this
+time describing the intent to do user research and a brainstorming session,
+https://docs.google.com/document/d/1bjiw5L4E8FztRtfsy4Nu9A6qSP-BJlK6BBzl6Rea-Ds/edit#heading=h.b7qckyjgu28i.
+This brainstorm resulted in a list of some existing convenience functions
+languages have and proposed ones, like a `start` which takes the tracer from the
+active span in the current context, and proposals for defining annotations for
+easier instrumentation.
+
+### Current challenges
+
+We know users can find the API and SDK to be complex and daunting, the challenge
+is to collect actionable feedback on this issues. That information is spread
+across various languages' issues, blog posts and other online commentary.
+Performing an end user survey will help collect some of this knowledge into a
+central place that is actionable.
+
+When developer experience issues are encountered and addressed they are
+currently done in the various language implementations individually and this can
+lead to drift between different implementations and with the spec. An example
+would be how to handle marking a code block as `untraced` which has
+implementations in at least Java, Ruby and Javascript, each done differently,
+named differently and with no specification. This means users working across
+languages experience different results for similar features and other language
+SIG's aren't necessarily even aware of the pain point that their users may
+actually be experiencing.
+
+### Goals, objectives, and requirements
+
+With the 3 main signals going stable the time is right to circle back and
+address issues users may have faced over the years they've now been using
+OpenTelemetry.
+
+This project aims to identify developer experience issues with using
+OpenTelemetry, to select 3 that are concerns which can be addressed through
+updates to the specification and resolve them.
+
+## Deliverables
+
+The first deliverable will be an end user survey. Assuming the agreement and
+participation of the End User SIG this will be done in cooperation with them. A
```

Comment:
I like the idea of doing end-user research, but it could be easier to start by investigating prior art and asking OTel contributors and authors of OTel instrumentation libraries about convenience they have to introduce.

To name a few:
- Java instrumentation API https://github.com/open-telemetry/oteps/pull/165
- (somewhat similar) [Micrometer Observation API](https://spring.academy/guides/microservices-observability-reactive-spring-boot-3) that intends to simplify multi-signal instrumentation
- Enriching auto-collected telemetry in user apps: https://github.com/open-telemetry/oteps/pull/207
- Reducing error noise (some error like 404/401 are not really errors) - https://github.com/open-telemetry/semantic-conventions/issues/1003

Reply:
Agree, I added a paragraph before this one that covers this now. Let me know what you think.